torch/fx/experimental/proxy_tensor.py

```diff
@@ -367,100 +422,32 @@ def __init__(self, tracer):
         self.sym_mode = ProxySymDispatchMode(tracer)
         self.trace_state = {}
 
-    def __torch_dispatch__(self, func_overload, types, args=(), kwargs=None):
+    def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         with self.sym_mode.enable(False):
-            return self.inner_torch_dispatch(func_overload, types, args, kwargs)
+            return self.inner_torch_dispatch(func, types, args, kwargs)
 
     @contextmanager
     def restore(self):
         with self.sym_mode.enable(True):
             with super().restore():
                 yield
 
-    def inner_torch_dispatch(self, func_overload, types, args=(), kwargs=None):
+    def inner_torch_dispatch(self, func, types, args=(), kwargs=None):
         if not self.enable_tracing:
-            return func_overload(*args, **kwargs)
+            return func(*args, **kwargs)
 
-        if symbolic_shapes.is_symbolic_op(func_overload):
+        if symbolic_shapes.is_symbolic_op(func):
             with self.restore():
-                return symbolic_shapes.handle_symbolic_op(func_overload, args, kwargs)
+                return symbolic_shapes.handle_symbolic_op(func, args, kwargs)
 
-        func = func_overload.overloadpacket
         # We don't want to convert torch.tensor constants into tracing objects.
-        if func_overload == aten.lift.default:
+        if func == aten.lift.default:
```

Comment:
Deleting this will cause tests to fail but I've forgotten why we do it this way :(

Reply:
So, I think I added it a while ago....

I think it's to handle cases like
```
aten::add(torch.tensor(0), 5)
```

Reply:
But I feel like your constant changes should handle this...

Lemme try removing it.

Reply:
Seems like it passes? LGTM :)